cypress/support/components/EditDialogModal.js

```diff
@@ -42,9 +42,10 @@ export function VerifyTags(tags) {
   })
 }
 
-export function AddTag(tag) { 
-  cy.Get('button.cl-tags__button-add#tags').Click()
-  cy.Get('input#tags').type(`${tag}{enter}`)
+export function AddTags(tags) { 
+  ClickAddTagButton()
+  if (!Array.isArray(tags)) tags = [tags]
+  tags.forEach(tag => {cy.Get('[data-testid="tags-input-tag-input"]').type(`${tag}{enter}`)})
```

Comment:
`cy.Get('[data-testid="tags-input-tag-input"]')` Was this needed?
I know it's better to use `data-testid`s than classes but I thought it should have worked without re-finding the DOM element for each tag. The component is supposed to preserve focus on the input after pressing `{enter}` to allow the user to enter multiple easily.

Reply:
Good catch, I was able to improve this. I've updated the code with the change.

Reply:
STILL TODO